lib/validatePeerDeps.js

```diff
@@ -30,6 +30,10 @@ module.exports = async () => {
         ],
         {
           cwd: cwd(),
+          // Sku's fixtures specify sku as a workspace dependency, resulting in an infinite symlink loop of
+          // sku/fixtures/*/node_modules/sku/fixtures/*/node_modules/...
+          // Setting the a depth stops it from trying to traverse this infinite loop and hang forever
```

Comment:
This is a great message in a bottle for the unfortunate soul that'll have to deal with this in the future 👍